charts/memgraph-high-availability/values.yaml

```diff
@@ -32,15 +32,25 @@ memgraph:
       logPVCClassName: ""
       logPVC: true
       logPVCSize: "256Mi"
-  affinity:
-    enabled: true
   ports:
     boltPort: 7687
     managementPort: 10000
     replicationPort: 20000
     coordinatorPort: 12000
   coordLoadBalancer:
     enabled: true
+  # Affinity controls the scheduling of the memgraph-high-availability pods, by disabling affinity, pods will be schedule on random nodes in the cluster that k8s scheduler chooses.
+  affinity:
+    enabled: true
```

Comment:
Resolved on the call, this can be removed. Default case is soft requirement so it should be deployed even when running minikube on a single node

Reply:
Fixed.